### PR TITLE
[zh] Resync HPA task

### PR DESCRIPTION
**Pull request by tengqm**
Related: #26178

### CODE REVIEW

File: content/zh/docs/tasks/run-application/horizontal-pod-autoscale.md [hunk 1]
```diff
@@ -434,6 +434,127 @@ usual.
...
+you should revise the HPA spec if that newly added container should also be used for
+scaling. If the specified container in the metric source is not present or only present in a subset
+of the pods then those pods are ignored and the recommendation is recalculated. See [Algorithm](#algorithm-details)
+for more details about the calculation. To use container resources for autoscaling define a metric
+source as follows:
+-->
+如果你更改缩放目标对象，令其使用新的、包含一组不同的容器的 Pod 规约，你就需要
+修改 HPA 的规约才能基于新添加的容器来执行规模扩缩操作。
+如果指标源中指定的容器不存在或者仅存在于部分 Pods 中，那么这些 Pods 会被忽略，
+HPA 会重新计算资源用量值。参阅[算法](#algorithm-details)小节了解计算细节。
```
Comment (ydFu) [inline content/zh/docs/tasks/run-application/horizontal-pod-autoscale.md:516]:
```suggestion
HPA 会重新计算资源用量值。参阅[算法](#algorithm-details)小节来了解更多计算细节。
```

**Review by tanjunchen — COMMENTED** [review]
/approve

**Pull request merged**